CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
+
+### Changed 
+* Corrected ml/stats/nodes permissions spelling from [@camAtGitHub](https://github.com/camAtGitHub)
+* Internal user usage clarification from [@reshippie](https://github.com/reshippie)
+* Remove `advanced_modules_enabled` examples from [@wjordan](https://github.com/wjordan)
+* CHANGELOG.md replaces release-notes
+
+### Security
+
+* Security patch for `org.cryptacular:cryptacular`
+* Security patch for `org.springframework:spring-core`
+* Security patch for `org.apache.kafka:kafka-clients`
+* Security patch for `org.springframework.kafka:spring-kafka-test`
+
+## [2.0.0-rc1] - 2022-04-20
+
+### Added
+
+* DLS privileges evaluation can process security roles from [@ch-govau](https://github.com/ch-govau)
+* Setting to enable role without DLS/FLS to override roles
+* DLS Term Lookup Queries
+
+### Changed 
+
+* Support for JDK17
+* Support for Gradle 7
+* Use standard Issue/Pull request templates from [@dblock](https://github.com/dblock)
+* Security configuration are placed side-by-side with other plugins
+* Cleaned up developer guide
+
+### Deprecated
+
+* Security tools deprecation
```

Comment:
this does not give users a lot of information:
* what even are "security tools" (e.g. just based on this name i'm unsure what this is about)
* why are they deprecated
* what is the replacement (if any)?
* is there a clear timeline by when this will be removed?